### PR TITLE
Correct tf.Gradient... to tf.train.Gradient...

### PR DESCRIPTION
**Pull request by JohnAllen**

### CODE REVIEW

**Comment by yifeif** [comment]
Thank you @JohnAllen! train.md is auto-generated, could you remove the change from PR?

**Comment by JohnAllen** [comment]
Happy to contribute!  Train.md removed from the PR.

**Comment by yifeif** [comment]
@tensorflow-jenkins test this please

**Comment by yifeif** [comment]
@tensorflow-jenkins test this please
